content/docs/start/data-and-model-versioning.md

```diff
@@ -82,35 +82,31 @@ outs:
     path: data.xml
 ```
 
-> \* See
-> [Large Dataset Optimization](/doc/user-guide/large-dataset-optimization) and
-> `dvc config cache` for more info. on file linking.
-
 </details>
 
 ## Storing and sharing
 
 You can upload DVC-tracked data or model files with `dvc push`, so they're
 safely stored [remotely](/doc/command-reference/remote). This also means they
 can be retrieved on other environments later with `dvc pull`. First, we need to
-setup a storage:
+setup a storage provider:
 
 ```dvc
 $ dvc remote add -d storage s3://mybucket/dvcstore
 $ git add .dvc/config
 $ git commit -m "Configure remote storage"
 ```
 
-> DVC supports the following remote storage types: Google Drive, Amazon S3,
-> Azure Blob Storage, Google Cloud Storage, Aliyun OSS, SSH, HDFS, and HTTP.
-> Please refer to `dvc remote add` for more details and examples.
+> DVC supports many remote storage types, including: Google Drive, Amazon S3,
+> Azure Blob Storage, Google Cloud Storage, Aliyun OSS, SSH, HDFS, and HTTP. See
+> `dvc remote add` for more details and examples.
 
 <details>
 
-### ⚙️ Set up a remote storage
+### ⚙️ Expand to set up a remote storage provider ☁
```

Comment:
I think "setup remote storage" though it doesn't sounds as nice is better since it's the term we have everywhere. Feel free to make an issue to discuss, decide, and change the terminology throughout though but the GS shouold prob be very consistent with existing terminology and "remote storage" is the most common form atm. Also, always happy to add some elegant emojis like here, but unfort remote storage isn't always on the cloud so it may be misleading.

Reply:
> "remote storage" is the most common form atm.

Also "DVC remote" and "data remote" (the last one at least should def. be revised IMO).

Reply:
changed